src/BigBlueButton.php

```diff
@@ -524,6 +549,21 @@ public function setJSessionId($jSessionId)
      * @throws \RuntimeException
      */
     private function processXmlResponse($url, $payload = '', $contentType = 'application/xml')
```

Comment:
`$contentType` should be always the same for xml request, or not?

Reply:
not always, see setConfigXML l. 213 vs createMeeting l.171
In both cases it is sending a payload, one time with xml and the other time as form-urlencoded